lerobot-web-be/app/services/calibrate.py

```
@@ -0,0 +1,43 @@
+from lerobot.teleoperators.so100_leader import SO100Leader, SO100LeaderConfig
+
+from ..models.calibrate import CalibrationParams, RobotKind
+
+enter_flag = False
+primary_file_descriptor, secondary_file_descriptor = pty.openpty()
+sys.stdin = os.fdopen(secondary_file_descriptor)
+
+
+def start_calibration(params: CalibrationParams):
+    robotId = params.robot_id
+    robotName = params.robot_name
```

Ruff (0.14.1)

Local variable `robotName` is assigned to but never used

Remove assignment to unused variable `robotName`

(F841)